manila/share/drivers/netapp/dataontap/cluster_mode/data_motion.py

```diff
@@ -369,7 +372,37 @@ def wait_for_quiesced():
                                              dest_volume,
                                              clear_checkpoint=False)
 
-    def break_snapmirror(self, source_share_obj, dest_share_obj, mount=True):
+    def check_snapmirror_health(self, source_share_obj, dest_share_obj):
+        """Check if SnapMirror relationship is in sync."""
+        snapmirror = self.get_snapmirrors(source_share_obj, dest_share_obj)[0]
+        if snapmirror.get('relationship-status') != 'idle':
```

Comment:
I think the expected state in happy path would be `quiesced`.
If quiesce did not finish in time and it was aborted, it goes into `idle`, I believe.
So we should accept both values for the relationship status.

Reply:
corrected